teams/3/styles.css

```diff
@@ -0,0 +1,846 @@
+/* Disregard these styles. This is ony for example purposes */
+
+h2 {
```

Comment:
please do not targe elements, use classes, I have mentioned several times, even for example, this is not accepted, you can remove the file too